Capri.Database/Entities/Proposal.cs

```diff
@@ -20,6 +18,7 @@ public class Proposal : IEntity
         [Required]
         public StudyLevel Level { get; set; }
         public StudyMode Mode { get; set; }
+        [Required]
```

Comment:
In case of cascade delete you should provide such information in DbContext class (onModelCreating method if i remember good).

Reply:
As far as I know, [Required] annotation should imply cascade deletion in this case